tests/test_models_dbi.py

```diff
@@ -10,6 +10,7 @@
 from qibo.quantum_info import random_hermitian
 
 NSTEPS = 50
+seed = 10
```

Comment:
```suggestion
SEED = 10
```
To follow the same notation as `NSTEPS`